packages/deployer/internal/ast-helper.js

```diff
@@ -0,0 +1,53 @@
+const { findAll } = require('solidity-ast/utils');
+
+function getSlotAddresses(contractName, ast) {
+  const contractNode = findContractNode(ast, contractName);
+  if (!contractNode) {
+    return null;
+  }
+
+  const slots = [];
+  for (const assignment of findAll('YulAssignment', contractNode)) {
+    if (assignment.variableNames[0].name === 'store.slot') {
```

Comment:
Note: I didn't find a better way to identify a namespace declaration than looking for the `store.slot := 0x...`

Reply:
Yeah, it's tricky. I'm sure we'll figure out a better way. I'd say at least we should add a ticket in the backlog to improve this.

Reply:
Done